### PR TITLE
update docs with renamed from parameter and me/feed query params

### PR DESCRIPTION
**Pull request by bendoyon**

### CODE REVIEW

**Comment by bendoyon** [comment]
Make sure to load the big diffs, it includes the missing params from 2 endpoints

File: docs/api-reference/spec/firework-v3-openapi.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     },
     "servers": [
         {
-            "url": "https://api.flare.io/firework/v3"
+            "url": "https://firework-api:5000/firework/v3"
```
Comment (aviau) [inline docs/api-reference/spec/firework-v3-openapi.json:10]: Thats probably bad?

Can you please generate it from prod instead?

Comment (bendoyon) [reply docs/api-reference/spec/firework-v3-openapi.json:10]: Ah my bad, I didn't notice this, sorry.

Comment (bendoyon) [reply docs/api-reference/spec/firework-v3-openapi.json]: Fixed.

**Review by aviau — APPROVED** [review]

**Pull request merged**